### PR TITLE
Fixes cybernetic eyeballs not actually protecting from welders

### PR DESCRIPTION
**Pull request by ghost**
There's a variable called welding_proof in eyes, that was never used in anything. By assumption it's only used for welder protection rather than protecting against flashes (Whichshould be eyeprot, considering what it's used in.)

Guess now you don't need goggles or helmets to weld.

### CODE REVIEW

File: code/game/objects/items/weapons/tools.dm [hunk 1]
```diff
@@ -453,6 +453,9 @@
 		var/datum/organ/internal/eyes/E = H.internal_organs_by_name["eyes"]
 		if(!E)
 			return
+		if(E.welding_proof)
+			usr.simple_message("<span class='notice'>Your eyelenses darken to accommodate for the welder's glow</span>")
```
Comment (9600bauds) [inline code/game/objects/items/weapons/tools.dm]: Aaaaaahhhh why is this whole proc using usr instead of user

Comment (9600bauds) [inline code/game/objects/items/weapons/tools.dm]: Also needs . at the end

Comment (ghost) [inline code/game/objects/items/weapons/tools.dm]: Should I change the stuff to user too?

**Comment by Exxion** [comment]
Do the cybernetic eyes you can have at roundstart have welding protection?
I think they should have _some_ bonus to make up for their EMP weakness, but that might be too much

**Comment by ghost** [comment]
That'll hopefully be addressed when we come to actually touching on cybernetics, once all the ideas are brought out before we set anything in stone

File: code/game/objects/items/weapons/tools.dm (hunk 1 above)
Comment (ghost) [inline code/game/objects/items/weapons/tools.dm]: So should I, or should I just change that full stop? @9600bauds

Comment (9600bauds) [inline code/game/objects/items/weapons/tools.dm]: If it's not too much of a bother, please change the 5 or so instances of usr to user in this proc

**Comment by ghost** [comment]
And that should be good now then

**Comment by 9600bauds** [comment]
:+1:

**Comment by ghost** [comment]
Also, been looking around the whole 'see in dark' stuff, and it is hardcoded in so many points in such silly ways, like 'if istype mob/living/carbon/human see_in_dark = 3'

**Comment by Exxion** [comment]
pain

**Comment by ghost** [comment]
yey

**Pull request merged**